src/TiciTacaToeyGameEngine.ts

```diff
@@ -129,6 +162,7 @@ class TiciTacaToeyGameEngine implements GameEngine {
           ) {
             reject({ error: ErrorCodes.GAME_ALREADY_IN_PROGRESS, message });
           }
+          // start the timer of the first player
```

Comment:
We should never start timers in the `validate` method, as the actual game object updates happen in the `transition` method